static/app/types/index.tsx

```diff
@@ -453,10 +453,17 @@ export type EventsStats = {
   data: EventsStatsData;
   totals?: {count: number};
   order?: number;
+  start?: string;
+  end?: string;
 };
 
 // API response format for multiple series
-export type MultiSeriesEventsStats = {[seriesName: string]: EventsStats};
+export type MultiSeriesEventsStats = {
+  [seriesName: string]: EventsStats;
+} & {
+  start?: string;
+  end?: string;
```

Comment:
start/end should only exist on `EventsStats` now right?

Reply:
yup, removed from `MultiSeriesEventsStats`!